packages/app-store/cal-ai/config.json

```diff
@@ -14,5 +13,10 @@
   "isTemplate": false,
   "__createdUsingCli": true,
   "__template": "basic",
-  "dirName": "cal-ai"
+  "dirName": "cal-ai",
+  "paid": {
+    "price": 25,
+    "priceId": "price_1O1ziDH8UDiwIftkDHp3MCTP",
```

Comment:
Not a blocker: This should be configurable. Need to figure out where it should be read from exactly.

Reply:
wdym? Peer asked for it to be in the config

Reply:
I mean that in general while supporting paid apps, self hosters might want to configure the price id as it would be different for them. So, not an immediate requirement.

Reply:
Even when I was testing locally, I had to change it as per my stripe account and I have to remember to not commit that.